Source/Core/InputCommon/ControllerInterface/DInput/DInputKeyboardMouse.h

```diff
@@ -10,13 +10,23 @@
 #include "InputCommon/ControllerInterface/CoreDevice.h"
 #include "InputCommon/ControllerInterface/DInput/DInput8.h"
 
+
 namespace ciface::DInput
 {
 void InitKeyboardMouse(IDirectInput8* const idi8, HWND hwnd);
 
+extern void Save_Keyboard_and_Mouse_Settings();
+extern void Load_Keyboard_and_Mouse_Settings();
+
 using RelativeMouseState = RelativeInputState<Common::TVec3<LONG>>;
 void SetKeyboardMouseWindow(HWND hwnd);
 
+extern double cursor_sensitivity;  // 2 for full screen mapping
+extern unsigned char center_mouse_key;
```

Comment:
Please don't introduce more global state

Reply:
I wouldn't if I didn't have to. Do you have a suggestion on how to get rid of those? They're also not in the global namespace. They're in ciface::DInput::.